app/src/main/java/com/github/muellerma/coffee/tiles/TimeoutTile.kt

```diff
@@ -5,27 +5,17 @@ import android.content.Context
 import android.os.Build
 import android.util.Log
 import androidx.annotation.RequiresApi
-import com.github.muellerma.coffee.*
+import com.github.muellerma.coffee.ForegroundService
+import com.github.muellerma.coffee.ServiceStatus
+import com.github.muellerma.coffee.ServiceStatusObserver
+import com.github.muellerma.coffee.coffeeApp
 
 @RequiresApi(Build.VERSION_CODES.N)
 class TimeoutTile : AbstractTile() {
     override fun onClick() {
         Log.d(TAG, "onClick()")
-        val prefs = Prefs(applicationContext)
-        when {
-            coffeeApp().lastStatusUpdate is ServiceStatus.Stopped -> {
-                prefs.timeout = prefs.firstTimeout
-                ForegroundService.changeState(this, ForegroundService.Companion.STATE.START, false)
-            }
-            prefs.nextTimeout == 0 -> {
-                prefs.timeout = 0
-                ForegroundService.changeState(this, ForegroundService.Companion.STATE.STOP, false)
-            }
-            else -> {
-                prefs.timeout = prefs.nextTimeout
-            }
 
-        }
+        ForegroundService.toggleState(coffeeApp())
```

Comment:
I don't understand this PR. This tile is intended to go to the next timeout while the other tile is an on/off toggle. Both ways are requested by users, so I'd keep them both,

Reply:
well this PR combines both, you can tap on the tile to go to the next time out, after 1 second, it'll start the chosen timeout, if you keep tapping, it'll keep choosing the next timeout, this way users can have one tile in their quick settings menu, instead of two. or we can add a third tile to do this functionality and keep the original two as is

Reply:
I'd keep the both existing tiles as they are. Can you add this as a third tile?

Reply:
yeah sure, no problem